Client/src/components/TransactionList.vue

```diff
@@ -1,13 +1,158 @@
 <style>
     .transaction-list {
         background-color: #fbfbfb;
-        padding-left: 150px;
         height: 100vh;
+        width: 280px;
+        border-right: solid 1px #d4d4d4;
+        padding-left: 95px;
+        padding-top: 13px;
     }
+
+    .h3{
+    	font-family: lato;
+    	font-size: 18px;
+    	font-weight:600;
+    	color: #153b50;
+    }
+
+    .filters{
+    	padding-top: 30px;
+    }
+
+    .h6{
+    	font-family: lato;
+    	font-size: 10px;
+    	font-weight:700;
+    	color: #a1b1b9;
+    	padding-bottom: 15px;
+    	letter-spacing: .2px;
+    }
+
+     .actions{
+    	font-family: lato;
```

Comment:
We should set the font-family on the body. The goal of my next PR is to switch us over to SASS and make the CSS more DRY